src/View/Concert/_form.html.twig

```diff
@@ -1,27 +1,44 @@
 {# TODO : manage errors display #}
 
-<form method="post">
+<form class="addconcerts method=post" aria-describedby="formulaire ajout de concert">
```

Comment:
No, the class quotes are not closed, and the ones for the method are not opened.